pkg/driver/driver.go

```diff
@@ -150,9 +150,18 @@ func WithEndpoint(endpoint string) func(*DriverOptions) {
 	}
 }
 
+func WithExtraTags(extraTags map[string]string) func(*DriverOptions) {
+	return func(o *DriverOptions) {
+		o.extraTags = extraTags
+	}
+}
+
 func WithExtraVolumeTags(extraVolumeTags map[string]string) func(*DriverOptions) {
 	return func(o *DriverOptions) {
-		o.extraVolumeTags = extraVolumeTags
+		if o.extraTags == nil && extraVolumeTags != nil {
```

Comment:
Ideally we'd have 2 checks: 1 for printing the warning if `extraVolumeTags != nil` and the other to overwrite `o.extraTags`. But I think this is fine for now.